src/app/components/global/ArrowBtn.jsx

```diff
@@ -0,0 +1,9 @@
+import styles from "./ArrowBtn.module.scss";
+
+export default function ArrowBtn({ direction, onClick }) {
+  return (
+    <button className={`${styles.arrow} ${styles[direction]}`} onClick={onClick}>
+      arrow {direction}
```

Comment:
Hvorfor står der "arrow"?

Reply:
Det er fordi der skal være en tekst i et button element, så jeg fik teksten til at være hvilken slag pil det er, men gjorde teksten usynlig. Så hvis skærmlæsere fik den læst højt, så håbede jeg den ville læse arrow left eller arrow right op for brugeren

Reply:
Okay, jeg tror faktisk at det er hér, at vi skal bruge aria-labels, men tror lige vi skal snakke om det :-)